src/renderer/services/dex.js

```diff
@@ -201,9 +201,10 @@ export default {
     });
   },
 
-  getTradeHistoryBars(tradeHistory, resolution, from, to) {
+  getTradeHistoryBars(tradeHistory, resolution, from, to, last) {
     const bars = [];
-    const trades = tradeHistory.trades;
+    const trades = tradeHistory.trades.slice(0);
+    trades.reverse();
```

Comment:
Websocket is pushing trades into the array using unshift. Should probably just reverse websocket to push them in at the end instead, this will require reversing them also when fetchTradeHistory in dex.js is called (such as when websocket reconnects or market changes), but it saves needing to unshift each time a trade comes in from websocket. Also, that means it probably requires iterating in opposite order when filling the recent trades list in the UI.